README.md

```diff
@@ -68,8 +68,24 @@ This will create a bunch of static files in `/build`.
 This project is re-deployed by a Jenkins task every hour (to pick up external
 changes). It is [hosted on S3][terraform].
 
+## Pre-commit hooks
```

Comment:
Is it possible to make all of this available on the dev VM, or do you have to run `pre-commit install` per project?

Reply:
They're linked to git config, so I think you would have to install for each project where the `.pre-commit-config.yaml` file exists.

Reply:
Ah ok. I think the git hook is still more useful than a CI job would be. Should we add some linux installation instructions here as well?

Reply:
Will do.